ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -171,13 +228,22 @@ class IndividualRow extends React.Component
 
 export { IndividualRow as IndividualRowComponent }
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state, ownProps) => ({
   project: getProject(state),
   samplesByGuid: getSamplesByGuid(state),
+  matchmakerSubmission: getMatchmakerSubmissions(state)[ownProps.family.projectGuid][ownProps.individual.individualId],
 })
 
-const mapDispatchToProps = {
-  updateIndividual,
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+  return {
+    updateIndividual: (values) => {
+      return dispatch(updateIndividual(values))
+    },
+    deleteMmePatient: () => {
+      return dispatch(dispatchDeleteMmePatient(ownProps.individual))
```

Comment:
seems better to do `dispatch(deleteMmePatient` instead of `dispatch(dispatchDeleteMmePatient`